rest-server/src/config/express.js

```diff
@@ -44,17 +46,16 @@ app.use('/api/v1', router);
 
 // catch 404 and forward to error handler
 app.use((req, res, next) => {
-  const err = new Error('API not found');
-  err.status = 404;
-  next(err);
+  next(createError('Not Found', 'ERR_NO_API', `API ${req.url} not found.`));
 });
 
 // error handler
 app.use((err, req, res, next) => {
-  logger.warn('%s', err.stack);
+  logger.warn(err.stack);
   res.status(err.status || 500).json({
+    code: err.code,
     message: err.message,
-    error: config.env === 'development' ? err.stack : '',
```

Comment:
>error [](start = 4, length = 5)

If we removed error here, will it cause client incompatible?

Reply:
`Error#stack`, like comment, is listed mostly just for developer use and it is useless for automatic tools, and it has no content in `production` environment, so any client compatible with production environment should not depend on this field.